webview/src/plots/components/Plots.tsx

```diff
@@ -88,7 +90,8 @@ const PlotsContent = () => {
   }
 
   return (
-    <div ref={wrapperRef}>
+    <div ref={wrapperRef} className={styles.plotsContent}>
+      <TooManyPlots />
```

Comment:
[Q] Is it a normal pattern to have the band at the bottom of the screen instead of the top?

Reply:
It does not really matter. At first, I was building it to be at the top and decided to move it down later. I'll move it in the tree, which will help with reading the code.